### DISCUSSION TITLE
How to add motif logo in annotation file?

### DISCUSSION THREAD

**Discussion by ljljolinq1010** [aertslab/pycistarget#15] [Q&A] [answered]
Hi, I am using run_pycistarget for zebrafish analysis, I have generated my custom motif-to-TF annotation file, I am wondering how can I add motif logo information?

**Comment by SeppeDeWinter** [comment, marked as answer]
Hi @ljljolinq1010 

Sorry for the late reply.

As long as the motif-to-TF annotation file follows the convention of the motif-to-TF annotation files we host for human, mouse and fly (see: https://resources.aertslab.org/cistarget/motif2tf/). And as long as you don't have any motifs which are not present in our collection, the logo's will be automatically pulled from our server. For an example on how this is done, see: https://github.com/aertslab/pycistarget/blob/0ed8289bda8feb8b137f243864213cee6b06d7f0/pycistarget/motif_enrichment_cistarget.py#L425 

Best,

Seppe

**Reply by ljljolinq1010** [reply]
I see, thanks a lot.
